### PR TITLE
Cloud app backup

### PR DESCRIPTION
**Pull request by Rohit-PX**
**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
Stork integration tests for application backups will be able to run on the GCE platform

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
no

### CODE REVIEW

File: test/integration_test/applicationbackup_test.go [hunk 1]
```diff
@@ -1053,3 +1059,16 @@ func deleteApplicationRestoreList(appRestoreList []*storkv1.ApplicationRestore)
 	logrus.Info("Deleted all restores")
 	return nil
 }
+
+func getBackupLocationForVolumeDriver(volumeDriver string) (storkv1.BackupLocationType, error) {
```
Comment (disrani-px) [inline test/integration_test/applicationbackup_test.go]: Good for now, but we should put this in the driver in a separate change

File: test/integration_test/common_test.go [hunk 2]
```diff
@@ -124,18 +128,24 @@ func setup() error {
 
 	}
 	logrus.Infof("Using provisioner: %s", provisioner)
-	if err = schedulerDriver.Init(scheduler.InitOptions{
-		SpecDir:             "specs",
+	var customAppConfig map[string]scheduler.AppConfig
+	// For non-PX backups use default-scheduler in apps instead of stork
+	if provisioner != "pxd" {
```
Comment (disrani-px) [inline test/integration_test/common_test.go]: Same as the default backup location should put this in the driver in a separate change

**Comment by Rohit-PX** [comment]
```    --- PASS: TestApplicationBackup/applicationBackupRestoreTest (0.00s)
    --- PASS: TestApplicationBackup/preExecRuleTest (0.00s)
    --- PASS: TestApplicationBackup/postExecRuleTest (0.00s)
    --- PASS: TestApplicationBackup/preExecMissingRuleTest (0.00s)
    --- PASS: TestApplicationBackup/postExecMissingRuleTest (0.00s)
    --- PASS: TestApplicationBackup/preExecFailingRuleTest (0.00s)
    --- PASS: TestApplicationBackup/postExecFailingRuleTest (0.00s)
    --- PASS: TestApplicationBackup/labelSelector (0.00s)
    --- PASS: TestApplicationBackup/scheduleTests (2044.64s)
        --- PASS: TestApplicationBackup/scheduleTests/intervalTest (460.38s)
        --- PASS: TestApplicationBackup/scheduleTests/dailyTest (456.57s)
        --- PASS: TestApplicationBackup/scheduleTests/weeklyTest (417.25s)
        --- PASS: TestApplicationBackup/scheduleTests/monthlyTest (426.92s)
        --- PASS: TestApplicationBackup/scheduleTests/invalidPolicyTest (280.50s)
```

File: test/integration_test/applicationbackup_test.go (hunk 1 above)
Comment (Rohit-PX) [reply test/integration_test/applicationbackup_test.go]: will send separate PR in torpedo

File: test/integration_test/common_test.go (hunk 2 above)
Comment (Rohit-PX) [reply test/integration_test/common_test.go]: will send separate PR in torpedo.

**Pull request merged**